sdk/batch/microsoft-azure-batch/pom.xml

```diff
@@ -71,7 +71,7 @@
     <dependency>
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-core</artifactId>
-      <version>2.13.5</version> <!-- {x-version-update;com.fasterxml.jackson.core:jackson-core;external_dependency} -->
+      <version>2.15.4</version> <!-- {x-version-update;com.fasterxml.jackson.core:jackson-core;external_dependency} -->
```

Comment:
TODO: Add special version for `microsoft-azure-batch` as it supports Java 7 and Jackson Core requires Java 8 starting with version 2.14.0